### PR TITLE
release/v2.2007 Fix integer overflow on 32-bit architectures

### PR DESCRIPTION
**Pull request by jsoriano**
Similar to #1541, but for v2.2007 branch.

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/badger/1558)

### CODE REVIEW

File: value.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ import (
 
 // maxVlogFileSize is the maximum size of the vlog file which can be created. Vlog Offset is of
 // uint32, so limiting at max uint32.
-var maxVlogFileSize = math.MaxUint32
+var maxVlogFileSize uint32 = math.MaxUint32
```
[the remark this answers is not in the document]
Comment (jsoriano) [reply value.go:50]: Global variable not introduced in this PR, and is still a global variable [in master branch](https://github.com/dgraph-io/badger/blob/e3a0d29f75d9978d8230956c39f0dbdde2f1cb63/value.go#L41). I would keep it as is here.

[the remark this answers is not in the document]
Comment (jarifibrahim) [reply value.go:50]: We can ignore the comment by codelingo.

**Review by jarifibrahim — APPROVED** [review]

**Comment by jarifibrahim** [comment]
Thanks @jsoriano

**Comment by g1ibby** [comment]
@jsoriano Thanks! You helped me so much. 
@jarifibrahim I would like to clarify when these changes will be included in the new release?

**Comment by jarifibrahim** [comment]
Hey @g1ibby , we don't have the next patch for v2.2007 planned. Is it possible for you to use the latest release? v3.2011.1?

**Comment by g1ibby** [comment]
okay, thank you for the information. I'll be thinking about using v3.*

**Pull request merged**